client/jetpack-connect/site-topic.js

```diff
@@ -26,7 +28,15 @@ class JetpackSiteTopic extends Component {
 
 		this.props.saveSiteVertical( siteId, siteVertical );
 
-		page( `/jetpack/connect/plans/${ siteSlug }` );
+		const jpVersion = this.props.siteJetpackVersion;
+		if ( ! jpVersion ) {
+			return null;
+		}
+		if ( versionCompare( jpVersion, 7.2 ) < 0 || jpVersion === '7.2-alpha' ) {
```

Comment:
This seems off. What about the beta version? I suggest that we change it to just:

```
versionCompare( jpVersion, '7.2-alpha', '>=' )
```